chapter_linear-networks/linear-regression-concise.md

```diff
@@ -406,7 +406,7 @@ print('b的估计误差：', true_b - b)
 
 ## 练习
 
-1. 如果将小批量的总损失替换为小批量损失的平均值，你需要如何更改学习率？
+1. 如果用`nn.MSELoss(reduction='sum')`替换`nn.MSELoss()`，你需要如何更改学习率？
```

Comment:
```suggestion
:begin_tab:`pytorch`
1. 如果用`nn.MSELoss(reduction='sum')`替换`nn.MSELoss()`，你需要如何更改学习率？
:end_tab:
```

Reply:
Great suggestion @gfhe ! Could you also add two separate tabs for `mxnet` and `tensorflow`?